lib/firespring_dev_commands/templates/aws.rb

```diff
@@ -120,6 +120,57 @@ def create_eol_task!
           end
         end
       end
+
+      # rubocop:disable Metrics/MethodLength
+      # Create the rake task for the hosted zone method
+      def create_hosted_zone_task!
```

Comment:
I would make this method name more clear - and consider breaking up into two methods (create_hosted_zone_dns_logging_activate / deactivate maybe)